Resources/config/services.xml

```diff
@@ -11,10 +11,8 @@
 
     <services>
 
-        <service id="jcroll_foursquare_api.foursquare_client"
-                 class="Jcroll\FoursquareApiClient\Client\FoursquareClient"
-                 factory-class="Jcroll\FoursquareApiClient\Client\FoursquareClient"
-                 factory-method="factory">
+        <service id="jcroll_foursquare_api.foursquare_client" class="Jcroll\FoursquareApiClient\Client\FoursquareClient">
+            <factory class="Jcroll\FoursquareApiClient\Client\FoursquareClient" method="factory" />
```

Comment:
Hi @trq thanks for your contribution.

I believe this pattern was introduced in symfony 2.6 so below it should be:
`"symfony/framework-bundle": "~2.6|^3.0"`

After that we are good for merge and much appreciated.